BreakinIn/Messages/ReptIn.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BreakinIn.Messages
+{
+    public class ReptIn : AbstractMessage
+    {
+        public override string _Name { get => "rept"; }
+
+        public string PERS { get; set; }
+        public string PROD { get; set; }
+        public string LANG { get; set; }
+		
+		public override void Process(AbstractEAServer context, EAClient client)
```

Comment:
This indentation does not make any sense. A quick way of getting visual studio to automatically do your indentation for you is to cut and paste the function in place.

If you are using another editor, set it to use 4 spaces instead of tabs.